packages/lib-classifier/src/components/Classifier/components/Layout/helpers/getLayout/getLayout.js

```diff
@@ -0,0 +1,11 @@
+import DefaultLayout from '../../components/DefaultLayout'
+
+const layouts = {
+  default: DefaultLayout
+}
+
+function getLayout (layout) {
+  return layouts[layout] || null
```

Comment:
Perhaps the fall back here should be the default layout rather than null. In the unlikely chance that someone happens to set the layout prop to null, I still think the default layout should render.

Reply:
Cool - I'll do that and make it log a warning as well.